test/rest-smoketest.sh

```diff
@@ -200,13 +212,28 @@ dotest "204::*::*" "${obj1_vers1}" -X DELETE
 dotest "404::*::*" "${obj1_vers1}"
 dotest "409::*::*" /ns-${RUNKEY}/foo2/obj1
 
-# test chunk upload
-cat > ${TEST_DATA} <<EOF
-{"chunk_bytes": 1024,
- "total_bytes": ${script_size},
- "content_type": "application/x-bash",
- "content_md5": "$md5"}
+total_bytes=${script_size}
+chunk_bytes=1024
+upload_file_name="$0"
+
+# test chunk upload (S3 requires at least 5MB chunks)
+if [[ $DEPLOYMENT == "amazons3" ]]
+then
+    upload_file_name="/tmp/dummy-${RUNKEY}"
+    chunk_bytes=5242881
+    # generate 5MB + file
+    dd if=/dev/urandom bs=${chunk_bytes} count=1 2>/dev/null | base64 > ${upload_file_name}
```

Comment:
This generated file will be larger than chunk_bytes due to inflation of base64 encoding... you need to either truncate it or set chunk_bytes to the true size, I think...?